src/librustc_session/config.rs

```diff
@@ -494,7 +494,7 @@ impl Input {
 #[derive(Clone, Hash)]
 pub struct OutputFilenames {
     pub out_directory: PathBuf,
-    filestem: String,
+    pub filestem: String,
```

Comment:
Why this change?

Reply:
Good catch - sorry this was a mistake. Reverted.